src/components/Link.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
```

Comment:
Do not introduce a new directory - if you want to make a new organization it should be done in a separate PR.